src/modules/ens/selectors.spec.ts

```diff
@@ -184,3 +192,65 @@ describe('when using isWaitingTxSetAddress selector', () => {
     })
   })
 })
+
+describe('when using getContributableNames selector', () => {
+  let contributableNames: ENSState['contributableNames']
+  beforeEach(() => {
+    contributableNames = {
+      'test.dcl.eth': {
+        name: 'test.dcl.eth'
+      } as ENS,
+      'test2.eth': {
+        name: 'test2.eth'
+      } as ENS
+    }
+    state = {
+      ...state,
+      ens: {
+        ...state.ens,
+        contributableNames
+      }
+    }
+  })
+
+  it('should return a record of contributable names', () => {
+    expect(getContributableNames(state)).toEqual(contributableNames)
+  })
+})
+
+describe('when using getContributableNamesError selector', () => {
```

Comment:
It seems that there are some articles missing, would you mind adding them?
```suggestion
describe('when using the getContributableNamesError selector', () => {
```
The same goes for other its and describes in the file.